packages/api/src/controllers/sensorController.js

```diff
@@ -149,6 +151,7 @@ const sensorController = {
             registeredSensors,
           });
         } else {
+          sendSensorNotification(user_id, farm_id, SensorNotificationTypes.SENSOR_BULK_UPLOAD);
```

Comment:
I would remove this, since the async sensor upload logic has not been added yet, and I don't think the actually calling of this function is in scope of this ticket.